python/cucim/src/cucim/skimage/filters/tests/test_median.py

```diff
@@ -6,6 +6,7 @@
 
 from cucim.skimage._shared.testing import expected_warnings
 from cucim.skimage.filters import median
+from cucim.skimage import morphology
```

Comment:
Think the style check wants these reordered

```suggestion
from cucim.skimage import morphology
from cucim.skimage._shared.testing import expected_warnings
from cucim.skimage.filters import median
```